test/hotspot/jtreg/runtime/cds/appcds/TestDumpClassListSource.java

```diff
@@ -117,9 +118,28 @@ public static void main(String[] args) throws Exception {
         checkFileExistence("Archive", fileArchive);
         checkFileExistence("ClassList", fileList);
 
-        output.shouldHaveExitValue(0);
+        output.shouldHaveExitValue(0)
+              .shouldContain(warningMessage);
```

Comment:
Why would the above process generate the "Skipping java/lang/invoke/BoundMethodHandle$Species_" message?

Reply:
This is dynamic dump, the problem is filed against static dump. When do dynamic dump we did not have a those in SPECIES_RESOLVE in classlist like static dump. They are not regenerated in fact.